docs/learning/backend/Workspace.md

```diff
@@ -1,475 +1,226 @@
+To explore [Workspace]($backend) concepts, let's take the example of an imaginary application called "LandscapePro" that allows users to decorate an iModel by adding landscaping features like trees, shrubs, flower beds, and patio furniture.
 
-When an iTwin.js backend starts, [IModelHost.startup]($backend) creates an instance of a [Workspace]($backend), in [IModelHost.appWorkspace]($backend).
-
-`IModelHost.appWorkspace` can be used to customize the session according to the choices of the application, including the default values for its settings.
-
-Whenever an application opens an iModel using the [IModelDb]($backend) class, it creates an instance of a [Workspace]($backend) in [IModelDb.workspace]($backend) to customize the session according to the choices made by administrators for the iTwin and the iModel.
-
-When combined, the `IModelHost.appWorkspace` and the `IModelDb.workspace` customize a session according to the:
-
-1. application's defaults
-2. organization of the user
-3. current iTwin
-4. current iModel
-
-In the list above, later entries tend to change more frequently and, in the case of duplicate values, later entries override earlier ones.
-
-[Workspace]($backend)s expresses the current state of the session in two forms:
+## Settings
 
-  1. [Settings](#settings)
-  2. [WorkspaceDb](#workspaceDbs)
+[Settings]($backend) are how administrators of an application or project configure the workspace for end-users. Be careful to avoid confusing them with "user preferences", which can be configured by individual users. For example, an application might provide a check box to toggle "dark mode" on or off. Each individual user can make their own choice as to whether they want to use this mode - it is a user preference, not a setting. But an administrator may define a setting that controls whether users can see that check box in the first place.
 
-`Settings` are *named parameters* that an application defines and whose values are supplied at runtime. `WorkspaceDb`s hold *named resources* (i.e. data) that the application uses. `Settings` and `WorkspaceDb`s are often related in application logic, e.g.:
+A [Setting]($backend) is simply a name-value pair. The value can be of one of the following types:
+- A `string`, `number`, or `boolean`;
+- An `object` containing properties of any of these types; or
+- An `array` containing elements of one of these types.
 
-- a Setting may contain the "formula" to find a resource
-- a `WorkspaceDb`s may hold a resource that defines a group of `Settings`
+A [SettingName]($backend) must be unique, 1 to 1024 characters long with no leading nor trailing whitespace, and should begin with the schema prefix of the [schema](#settings-schemas) that defines the setting. (More on schemas shortly). For example, LandscapePro might define the following settings:
```

Comment:
Not sure the part "more on schemas shortly" is even needed, since "schema" is a link. But if you want to keep that, I suggest changing it to this:

```suggestion
A [SettingName]($backend) must be unique, 1 to 1024 characters long with no leading nor trailing whitespace, and should begin with the schema prefix of the [schema](#settings-schemas) that defines the setting - more on schemas shortly. For example, LandscapePro might define the following settings:
```

Reply:
removed.